src/protocols/dhcpv6/encode.c

```diff
@@ -52,18 +52,13 @@ static ssize_t encode_tlv_hdr(uint8_t *out, size_t outlen,
 
 static inline bool is_encodable(fr_dict_attr_t const *root, VALUE_PAIR const *vp)
 {
-	if (!vp) return false;
-	if (vp->da->flags.internal) return false;
-	if (!fr_dict_parent_common(root, vp->da, true)) return false;
-
 	/*
 	 *	False bools are represented by the absence of
 	 *	an option, i.e. only bool attributes which are
 	 *	true get encoded.
 	 */
-	if ((vp->da->type == FR_TYPE_BOOL) && !vp->vp_bool) return false;
-
-	return true;
+	return vp && !vp->da->flags.internal && fr_dict_parent_common(root, vp->da, true) &&
```

Comment:
similar comments here.  4 lines of `if A do B` are easier read than `if A1 &&  A2 && A3 && A4`